analysers/experiments/processDuration.py

```diff
@@ -14,18 +14,30 @@
 def createQuery(CassandraRDD, experimentID):
```

Comment:
@VincenzoFerme At trial level it was computed and labelled "all", so now it's included at exp level.